clients/cpp/tests/system/test_system_runner.py

```
@@ -167,13 +167,21 @@ def run_system_tests():
             sys.exit(result.returncode)
 
     finally:
-        # Kill the server process group
         print("Cleaning up...")
         for proc in procs:
             try:
                 os.killpg(os.getpgid(proc.pid), subprocess.signal.SIGTERM)
             except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)